plenty_api/api.py

```diff
@@ -298,6 +298,11 @@ def __plenty_api_request(self,
             if method.lower() == 'post':
                 raw_response = requests.post(endpoint, headers=self.creds,
                                              params=query, json=data)
+
+            if method.lower() == 'put':
+                raw_response = requests.put(endpoint, headers=self.creds,
+                                             params=query, data=data)
+
```

Comment:
just for your information, I will drop this commit during merge as I already have something similar on my tree.